package.json

```
@@ -2,19 +2,21 @@
   "name": "covmap",
   "version": "0.1.0",
   "description": "A web app to display large-scale epidemiological data and forecasts of the SARS-CoV-2 pandemic.",
-  "repository": "git@github.com:CovOpen/CovMapper.git",
+  "repository": "https://github.com/DanielHabenicht/CovMapper",
```

```suggestion
  "repository": "https://github.com/CovOpen/CovMapper",
```
Still need to change this